contributing.md

```diff
@@ -0,0 +1,56 @@
+# Rootstock Contribution Guidelines - RIF Relay
+Thank you for investing your time in contributing to RIF Relay! Any contribution you make will be reflected on the [RIF Relay GitHub page](https://github.com/rsksmart/rif-relay). Your efforts help improve the Rootstock and RIF ecosystem.
+
+## Understanding the Project
+RIF Relay is a secure transaction relay system that enables users to pay fees using ERC-20 tokens; it was inspired by the Gas Station Network (GSN) project. It comprises four main modules: RIF Relay Contracts, RIF Relay Common, RIF Relay Client, and RIF Relay Server, each with specific functionalities and roles within the system.
+Before contributing, we recommend familiarizing yourself with RIF Relay and it purpose. Please refer to the [`Readme.md`](https://github.com/rsksmart/rif-relay#readme) for detailed information. 
+
+## New Contributor
+To get an overview of the project, read the README in the respective RIF Relay module repository. Here are some resources to help you get started:
+- Read the Code of Conduct.
+- How to make a contribution: Follow the steps outlined below.
+- Where to go for help? Join the [Rootstock Open Discord](https://rootstock.io/discord) for community support.
+- Credits and Attribution: Contributors are acknowledged in the project documentation.
+
+### How do I make a contribution?
+
+**Reporting a Bug**
+1. To report a bug, submit an issue on the GitHub repository with:
+   - A clear and descriptive title.
+   - A concise and specific description of the bug.
+   - Steps to reproduce the issue.
+   - Expected behavior and what actually happens.
+   - Any relevant screenshots or error messages.
+
+**Submitting a Fix**
+1. Fork the repository.
+2. Create a new branch for your fix.
+3. Make your changes in the new branch.
+4. Commit your changes with a descriptive commit message.
+5. Push the branch to your fork.
+6. Submit a pull request against the main branch of the original repository.
+
+**Proposing New Features**
+1. Open an issue on the GitHub repository describing the feature.
+2. Discuss the feasibility and relevance of the feature with the project maintainers.
+3. Once approved, follow the steps for submitting a fix to implement the feature.
+
+**How to Submit an Issue**
```

Comment:
What's the difference between issue and fix?

Reply:
@antomor I followed the template that was given to me by Owanate.
But I think An issue is a report about a bug, enhancement, or other task related to the project. It's essentially a way to track work that needs to be done.

A fix is the actual code or documentation change that resolves an issue. This is usually done through a commit or a series of commits in the project's repository.

Reply:
@zdaodu-iov I see that the theoretical difference, but I don't get what's the difference in the described process. For what I see here, the process looks the same:
1. fork the repo
2. create a new branch
3. push the changes
4. create a PR

Isn't the process the same? Why do we need to have 2 sections? Sorry if I'm missing something

Reply:
Some people tend to get confused with it but then we can remove it if you find it less important

Reply:
Yes, please, I don't like repetitions, because usually, at some point, they get out of sync 😅

Reply:
Okay, edited it